components/tidb_query/src/rpn_expr/impl_string.rs

```diff
@@ -269,6 +269,34 @@ pub fn strcmp(left: &Option<Bytes>, right: &Option<Bytes>) -> Result<Option<i64>
     })
 }
 
+#[rpn_fn]
+#[inline]
+pub fn un_hex(arg: &Option<Bytes>) -> Result<Option<Bytes>> {
+    Ok(arg.as_ref().and_then(|bytes| {
+        let mut v = Vec::with_capacity(bytes.len() / 2);
+        let skip = if bytes.len() % 2 == 1 {
+            // Add a '0' to the front, if the length is not the multiple of 2
+            v.push(val(b'0')? << 4 | val(bytes[0])?);
```

Comment:
why not simply `val(bytes[0])` since leading 0 doesn't affect the value?

Reply:
I fixed it, thx